script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -222,52 +222,52 @@ A person who could see these lines shouldn't be anywhere.
 Cough, cough, cough.
 There was no one.
 It was the end of summer in a sea of grass.
-At this rate, I felt like I was going to disappear.
+I felt like I was going to disappear like that.
 <Page60>
 But before that.
 "You. You shouldn't be crouching down like that."
-It was a woman's voice that reached my ears clearly.
+A woman's voice reached my ears clearly.
 <Page61>
 "Eh.........?"
-"Don't you "Eh" me.
-You're too small to be seen while crouching in the grass like that. I was almost about to send you flying."
+"Don't you 'eh' me.
+You're too small to be seen while crouching in the grass like that. I almost sent you flying."
 <Page62>
 The woman wryly pointed at me.
 ......I was a little pissed.
 I was the fourth tallest in my class, so I didn't think I was that short.
 <Page63>
 "Who's going to get sent flying?"
 "Don't be ridiculous, that should be obvious.
-We were the only ones here, so if you counted me out then who's left?"
+We're the only ones here, so if you counted me out, who's left?"
 The woman crossed her arms and said confidently.
 <Page64>
-"Well, it must be fate for us to meet like this, so why don't you keep me company for a bit? My name is Aozaki Aoko, and you are?"
+"Well, it must be fate for us to meet like this, so why don't you keep me company for a bit? My name's Aozaki Aoko. And you?"
 <Page65>
-The woman reached out to me as casually as if she was a friend.
-I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and grabbed her hand back as if I were being sucked into it.
+The woman reached out to me casually as if we were friends.
+I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and grabbed her hand back as if being sucked into it.
```

Comment:
This is no good.
> 断る理由も見当たらなくて、僕は遠野志貴と名前をつげて、吸いこまれるように手のひらを握り返した。
I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her grasp suck me off the ground.

What he's saying is that her pulling his hand up is "suction-like". This is the best I could come up with for that.